RELEASE_NOTES.md

```diff
@@ -0,0 +1,5 @@
+**NOTE:** Due to restrictions imposed on ad-hoc signed apps, macOS users may need to run the following command in Terminal before launching the app. Replace `/path/to` with the folder containing `Refresher.app`.
+
+```
+xattr -d com.apple.quarantine /path/to/Refresher.app
+```
```

Comment:
This should also contain instructions on determining your architecture. This can easily be a point of confusion if not documented

Reply:
As I said in the prior review, adding a note here but keeping the download names the same makes more sense to me (and probably other Apple silicon and Intel users.) The difference between both should be apparent when downloading from GitHub - and it's something other repos supporting multiple architectures do when naming their artifacts. 

A table could go here to provide a way for users to identify what they need to download.

Another option (that would only work for macOS) is to combine the x64 and arm64 binaries into one. This would create a universal app that runs on all machines.

Reply:
i dont know how hard it is to create a universal app, but i think thats the best option here, since then users dont have to worry about downloading the wrong one

Reply:
Universal apps can be created easily with the command line - can be made a step with the build (slightly complicating the existing setup, but I can do some testing later.) Since the entire app is self-contained inside of the main executable, it will be like my instance of Refresh:

- Script that updates to the latest commits
- Builds for osx-x64 and osx-arm64
- Uses `lipo` to combine these and any dependent frameworks
- Runs the resulting binary